packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -455,6 +455,7 @@ export type ContinueStreamOptions = {
    * Suffix to inject after the buffered data, but before the close tags.
    */
   suffix: string | undefined
+  signal?: AbortSignal
```

Comment:
I'm going to add singal to this helper in an upcoming PR but it's unrelated to this change it seems. Was this a holdover from some other work?